blenderproc/python/postprocessing/PostProcessingUtility.py

```diff
@@ -275,6 +278,151 @@ def trim_redundant_channels(image: Union[list, np.ndarray]) -> Union[list, np.nd
     return image
 
 
+def semantic_segmentation_mapping(image: Union[List[np.ndarray], np.ndarray],
+                                  map_by: Union[str, List[str]],
+                                  default_values: Optional[Dict[str, int]]) \
+        -> Dict[str, Union[np.ndarray, List[np.ndarray], List[Dict[str, Any]]]]:
+
```

Comment:
Missing documentation, would be nice to have even if it is not exposed.

Reply:
agree.

Reply:
or should it be exposed? Then it's missing in the `__init__.py`

Reply:
No it shouldn't